erpnext/stock/doctype/pick_list/pick_list.py

```diff
@@ -22,10 +22,14 @@
 
 
 class PickList(Document):
+	def validate(self):
+		self.validate_delivery_date()
+
 	def before_save(self):
 		self.set_item_locations()
 
 	def before_submit(self):
+		self.set_item_locations()
```

Comment:
Why are we adding this one?

Reply:
by mistake i'll change and add a new commit